src/PropertySuggester/Suggesters/SimpleSuggester.php

```diff
@@ -114,14 +141,39 @@ protected function getSuggestions( array $propertyIds, array $idTuples, $limit,
 				'ORDER BY' => 'prob DESC',
 				'LIMIT'    => $limit,
 				'HAVING'   => 'prob > ' . floatval( $minProbability )
-				)
-			);
+			)
+		);
+
 		$this->lb->reuseConnection( $dbr );
 
 		return $this->buildResult( $res );
 	}
 
 	/**
+	 * @param string $dbType
+	 * @param int $count
+	 * @return string
+	 */
+	private function getWeightedPropSelect( $dbType, $count ) {
+		// The sum of the weights needs to be 2.
```

Comment:
Why does it need to be 2? I believe I understood, but the comment should explain.